packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -256,7 +256,14 @@ export async function openAuthSessionAsync(
   }
 }
 
-// @docsMissing
+/**
+ * Dismisses the current authentication session. On web, it will close the popup window associated with auth process.
+ *
+ * @return The `void` on successful attempt, or throws error, if dismiss functionality is not available.
```

Comment:
```suggestion
 * @return The `void` on the successful attempt or throws an error if dismiss functionality is not available.
```

We can remove the `,` usage and make this sentence a bit more clear.

Reply:
Pushed an update, also updated the `dismissBrowser` comment which I have based the comment before.